### ISSUE TITLE
Silktide May 24 accessibility findings

### ISSUE THREAD

**Issue by chrisrhymes** [FutureNorthants/northants-design-system#478]
- Homepage promoted links insufficient contrast on focus - Ensure controls clearly indicate when they are selected
- HeroBanner text without the box has insufficient contrast with the background. This is probably due to replacing the lazy image component with the custom ResponsiveImage component. Maybe add the background overlay to the hero rather than the responsive image component?
